lib/dispatcher/client-h1.js

```diff
@@ -249,7 +249,9 @@ class Parser {
           this.timeout = timers.setFastTimeout(onParserTimeout, delay, new WeakRef(this))
         } else {
           this.timeout = setTimeout(onParserTimeout, delay, new WeakRef(this))
-          this.timeout.unref()
+          if (typeof this.timeout?.unref === 'function') {
+            this.timeout.unref()
+          }
```

Comment:
Even if... why not just optional chaining?

```suggestion
          this.timeout?.unref()
```